src/Templates/TodoTemplate/Bit.TodoTemplate/App/TodoTemplate.App.csproj

```diff
@@ -12,7 +12,7 @@
         <ApplicationTitle>TodoTemplate</ApplicationTitle>
 
         <!-- App Identifier -->
-        <ApplicationId>com.bitplatform.TodoTemplate</ApplicationId>
+        <ApplicationId>com.bitplatform.Todo.Template</ApplicationId>
```

Comment:
Why? @msynk

Reply:
if we don't change it, the `TodoTemplate` is going to be replaced with the target project name which is meaningless:
`com.bitplatform.testApp`